Crashes on album creation (it seems)

**Describe the bug**

I'm getting a crash during uploading quite a lot of pictures I took during the years with multiple phones

**Desktop & Client versions (please complete the following information):**
 - Operating System: Ubuntu 18.04
 - Version: 2.0

**Additional context**

Stack trace:

```
panic: runtime error: invalid memory address or nil pointer dereference                                                                                                                                                               
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x872cbc]                                                                                                                                                               
                                                                                                                                                                                                                                      
goroutine 16 [running]:                     
github.com/gphotosuploader/google-photos-api-client-go/v2/media_items.GooglePhotosRepository.CreateManyToAlbum(0xa1fae0, 0xc0000aa060, 0xa219a0, 0xc00002c0b0, 0xc001baa6e0, 0x4c, 0xc001fcf400, 0x1, 0x1, 0x40fc18, ...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/media_items/photoslibray_repository.go:62 +0x31c
github.com/gphotosuploader/google-photos-api-client-go/v2/media_items.HttpMediaItemsService.CreateManyToAlbum(...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/media_items/service.go:52
github.com/gphotosuploader/google-photos-api-client-go/v2/media_items.HttpMediaItemsService.CreateToAlbum(0xa21be0, 0xc0014aa0c0, 0xa219a0, 0xc00002c0b0, 0xc001baa6e0, 0x4c, 0xc0068026c0, 0x213, 0xc000322050, 0x4e, ...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/media_items/service.go:44 +0x108
github.com/gphotosuploader/google-photos-api-client-go/v2.Client.UploadFileToAlbum(0xa236c0, 0xc001fcec40, 0xa23780, 0xc0014aa0d0, 0xa17a00, 0xc000f91c00, 0xa219a0, 0xc00002c0b0, 0xc001baa6e0, 0x4c, ...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/client.go:41 +0x16f
github.com/gphotosuploader/gphotos-uploader-cli/internal/task.(*EnqueuedUpload).addMediaToAlbum(0xc00017e700, 0xc000322050, 0x4e, 0x0, 0x0, 0xc007a8be90, 0xc007a8be90)
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/task/upload.go:70 +0xb3
github.com/gphotosuploader/gphotos-uploader-cli/internal/task.(*EnqueuedUpload).Process(0xc00017e700, 0xc000322050, 0x4e)
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/task/upload.go:38 +0x53
github.com/gphotosuploader/gphotos-uploader-cli/internal/worker.(*Worker).Start.func1(0xc0000d62d0)                                                                                                                      
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/worker/queue.go:139 +0x357                                          
created by github.com/gphotosuploader/gphotos-uploader-cli/internal/worker.(*Worker).Start                        
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/worker/queue.go:127 +0x3f
```

The same thing happened to me as well -- I'm on Ubuntu 20.04. I'm uploading a large library. The code that I was running was the prebuilt binary from github as of Dec 31. I had tried to build it from source, but one of the dependencies did not resolve (`github.com/bmatcuk/doublestar/v2`).  I'm using go1.15.6.
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x872cbc]

goroutine 42 [running]:
github.com/gphotosuploader/google-photos-api-client-go/v2/media_items.GooglePhotosRepository.CreateManyToAlbum(0xa1fae0, 0xc00157a4b0, 0xa219a0, 0xc0000a4010, 0xc00ae99900, 0x4c, 0xc000620960, 0x1, 0x1, 0x40fc18, ...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/media_items/photoslibray_repository.go:62 +0x31c
github.com/gphotosuploader/google-photos-api-client-go/v2/media_items.HttpMediaItemsService.CreateManyToAlbum(...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/media_items/service.go:52
github.com/gphotosuploader/google-photos-api-client-go/v2/media_items.HttpMediaItemsService.CreateToAlbum(0xa21be0, 0xc00035be00, 0xa219a0, 0xc0000a4010, 0xc00ae99900, 0x4c, 0xc001e20fc0, 0x213, 0xc000644780, 0x2b, ...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/media_items/service.go:44 +0x108
github.com/gphotosuploader/google-photos-api-client-go/v2.Client.UploadFileToAlbum(0xa236c0, 0xc00037cf60, 0xa23780, 0xc00035be10, 0xa17a00, 0xc0018086c0, 0xa219a0, 0xc0000a4010, 0xc00ae99900, 0x4c, ...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/client.go:41 +0x16f
github.com/gphotosuploader/gphotos-uploader-cli/internal/task.(*EnqueuedUpload).addMediaToAlbum(0xc000382980, 0xc000644780, 0x2b, 0x0, 0x0, 0xc008f8de90, 0xc008f8de90)
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/task/upload.go:70 +0xb3
github.com/gphotosuploader/gphotos-uploader-cli/internal/task.(*EnqueuedUpload).Process(0xc000382980, 0xc000644780, 0x2b)
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/task/upload.go:38 +0x53
github.com/gphotosuploader/gphotos-uploader-cli/internal/worker.(*Worker).Start.func1(0xc0000c8140)
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/worker/queue.go:139 +0x357
created by github.com/gphotosuploader/gphotos-uploader-cli/internal/worker.(*Worker).Start
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/worker/queue.go:127 +0x3f
```

This the tail end of the log -- I've removed a few characters from the pageToken just in case that gives access to my library

Version (downloaded from github on Dec 31, 2020)
```
$ gphotos-uploader-cli version
gphotos-uploader-cli v2.0.0
$ sha1sum gphotos-uploader-cli
cfb1ab6841a74975df0d54f01720005c370a89e6  gphotos-uploader-cli

```

log
```

2021/01/02 23:20:27 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVi0zdlN4b2xFeGlrM04wS242aGg3X1J4aGpwWTlVOEE4VVRMMGtRdXFXdm15ZnotMGFvTC1hdExEeXotMFQ3MkYtZlRORlFMMFNyQi1nWmVxYm5HYnNUdzlLdUZZN2FLVTUyU1NxNWFQMlpWQmdjQ08yUEI4aUhwUWVsSnBid19HMTI2T2JDdVhkeXgxd1o0SFpoN2RPT0tHdF8yGgA
2021/01/02 23:20:27 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnh3Y3BmYkZuT3RVWGJhNGtQZm9WZy14TGtJR21zOHRVTVZ1VS1Nd1J6bTdQR19hVm8taThjVTBVTFJTQzZ5VlN6amJoeU41ckx6OVJzQ0dMX1dfeWFwbl9PMGMyTUI5RlREc1lMd0JGSHkzUUxacFk3X2o0MGdOMnh0ZUlLRDJqczRIb2FSd0hwb004UnliZjZMeXZnTzBtRGtIGgA
2021/01/02 23:20:27 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVi0zdlN4b2xFeGlrM04wS242aGg3X1J4aGpwWTlVOEE4VVRMMGtRdXFXdm15ZnotMGFvTC1hdExEeXotMFQ3MkYtZlRORlFMMFNyQi1nWmVxYm5HYnNUdzlLdUZZN2FLVTUyU1NxNWFQMlpWQmdjQ08yUEI4aUhwUWVsSnBid19HMTI2T2JDdVhkeXgxd1o0SFpoN2RPT0tHdF8yGgA
2021/01/02 23:20:28 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVi0zdlN4b2xFeGlrM04wS242aGg3X1J4aGpwWTlVOEE4VVRMMGtRdXFXdm15ZnotMGFvTC1hdExEeXotMFQ3MkYtZlRORlFMMFNyQi1nWmVxYm5HYnNUdzlLdUZZN2FLVTUyU1NxNWFQMlpWQmdjQ08yUEI4aUhwUWVsSnBid19HMTI2T2JDdVhkeXgxd1o0SFpoN2RPT0tHdF8yGgA
2021/01/02 23:20:28 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVndNT0FRNzR4bWFWN1hQNHhTN2FqaVJlZDltUTJGd3RNQUF2clZ0WHpSTVhTcUpLSnRQTW0yd2czWEtPazBfbDV2bTQxb01JTFZaamFGOHRLS3VHQ1dDMFZHWElHTlVFbEMxemV2YXloeEtYV3V0N0xScUtTVkp2T3hSUm15NGFHcjkyVGtha285VGJmQWkzZmU2WGRNanluS050GgA
2021/01/02 23:20:28 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVi10cGpxZnROMmQ2TjFZS1BBanBBNFV4NkpHTllzd2w4UHFieFFmODNwUXhYV0czdExYdW1ZTGNxY2wzeGN1Sk52QWd5QjdXLVBRSkppWGVMbHItQ0h5Y0dRY0d1WmtyYV9HRDVpZUs5ZVZCU2ZvRFp0cG5BdGRJbjNJbHBxb052a3RUQnN5NnlyTDFJSUd1NUs0MlZxTjhvUWF5GgA
2021/01/02 23:20:28 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnd6RzUwRlJhWDE5eENraEllZ0tvV1lDV0h5VVFjWWtYYTlxMDhmSnhlWkVjVXlMbjdQTC1ldkhnRlBLNExDcXBIZEVOSDdtMEt1YVNTUTNpQmxVVXVBVTRDSW5iWHBNTUNiNTdPM01ORUc4UW9ZUExmc2F2U1RrWHdoQU9BQlFMc2d0ZGJwS2RvMlVaVUdBMHplSlM3TTkzanlXGgA
2021/01/02 23:20:28 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnd6RzUwRlJhWDE5eENraEllZ0tvV1lDV0h5VVFjWWtYYTlxMDhmSnhlWkVjVXlMbjdQTC1ldkhnRlBLNExDcXBIZEVOSDdtMEt1YVNTUTNpQmxVVXVBVTRDSW5iWHBNTUNiNTdPM01ORUc4UW9ZUExmc2F2U1RrWHdoQU9BQlFMc2d0ZGJwS2RvMlVaVUdBMHplSlM3TTkzanlXGgA
2021/01/02 23:20:28 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnd6RzUwRlJhWDE5eENraEllZ0tvV1lDV0h5VVFjWWtYYTlxMDhmSnhlWkVjVXlMbjdQTC1ldkhnRlBLNExDcXBIZEVOSDdtMEt1YVNTUTNpQmxVVXVBVTRDSW5iWHBNTUNiNTdPM01ORUc4UW9ZUExmc2F2U1RrWHdoQU9BQlFMc2d0ZGJwS2RvMlVaVUdBMHplSlM3TTkzanlXGgA
2021/01/02 23:20:28 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjF0QTgteGpzS0o4eUJDSnRXYWttdnZKWmpMdGU3RTV3STJxNjBaVUFRZ2xoNzNtSU1EcW5lSWFkRGZZVkJVYWhOVmUyQzNuQ2hZREVPZE1uV1RiSkpfQklyd3gxMjFvRTI2NDEwRWxEZ2VHZXpwVkxtVzhnWVc2cUI1RXdXQzNuZDJhMlEyTy1rR25rLWd0dWprN2hHMGVvZ056GgA
2021/01/02 23:20:29 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVndNT0FRNzR4bWFWN1hQNHhTN2FqaVJlZDltUTJGd3RNQUF2clZ0WHpSTVhTcUpLSnRQTW0yd2czWEtPazBfbDV2bTQxb01JTFZaamFGOHRLS3VHQ1dDMFZHWElHTlVFbEMxemV2YXloeEtYV3V0N0xScUtTVkp2T3hSUm15NGFHcjkyVGtha285VGJmQWkzZmU2WGRNanluS050GgA
2021/01/02 23:20:29 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnh3Y3BmYkZuT3RVWGJhNGtQZm9WZy14TGtJR21zOHRVTVZ1VS1Nd1J6bTdQR19hVm8taThjVTBVTFJTQzZ5VlN6amJoeU41ckx6OVJzQ0dMX1dfeWFwbl9PMGMyTUI5RlREc1lMd0JGSHkzUUxacFk3X2o0MGdOMnh0ZUlLRDJqczRIb2FSd0hwb004UnliZjZMeXZnTzBtRGtIGgA
2021/01/02 23:20:29 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnh3Y3BmYkZuT3RVWGJhNGtQZm9WZy14TGtJR21zOHRVTVZ1VS1Nd1J6bTdQR19hVm8taThjVTBVTFJTQzZ5VlN6amJoeU41ckx6OVJzQ0dMX1dfeWFwbl9PMGMyTUI5RlREc1lMd0JGSHkzUUxacFk3X2o0MGdOMnh0ZUlLRDJqczRIb2FSd0hwb004UnliZjZMeXZnTzBtRGtIGgA
2021/01/02 23:20:29 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnh3Y3BmYkZuT3RVWGJhNGtQZm9WZy14TGtJR21zOHRVTVZ1VS1Nd1J6bTdQR19hVm8taThjVTBVTFJTQzZ5VlN6amJoeU41ckx6OVJzQ0dMX1dfeWFwbl9PMGMyTUI5RlREc1lMd0JGSHkzUUxacFk3X2o0MGdOMnh0ZUlLRDJqczRIb2FSd0hwb004UnliZjZMeXZnTzBtRGtIGgA
2021/01/02 23:20:29 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjl3RloyS1k5YnR6VmhpaUdXcjl4VGdJQ2hsTU5CY1BKWUlkRVNwVTZUNExLUWVKRHpGVzRiWXFEc3g5YUxISmJzZ0JtdVpMS3hmWFN5dGxkUmlqNkxPcTB5NzlCRFFQNmt2Zm1mNU9lbTYwNkExRXEzZ0RBZy1PZkNodmtuQnNfc1B1TWZNT1dYcGVvbkw4TVNRRllmUFFNTE1VGgA
2021/01/02 23:20:29 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVi10cGpxZnROMmQ2TjFZS1BBanBBNFV4NkpHTllzd2w4UHFieFFmODNwUXhYV0czdExYdW1ZTGNxY2wzeGN1Sk52QWd5QjdXLVBRSkppWGVMbHItQ0h5Y0dRY0d1WmtyYV9HRDVpZUs5ZVZCU2ZvRFp0cG5BdGRJbjNJbHBxb052a3RUQnN5NnlyTDFJSUd1NUs0MlZxTjhvUWF5GgA
2021/01/02 23:20:29 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjF0QTgteGpzS0o4eUJDSnRXYWttdnZKWmpMdGU3RTV3STJxNjBaVUFRZ2xoNzNtSU1EcW5lSWFkRGZZVkJVYWhOVmUyQzNuQ2hZREVPZE1uV1RiSkpfQklyd3gxMjFvRTI2NDEwRWxEZ2VHZXpwVkxtVzhnWVc2cUI1RXdXQzNuZDJhMlEyTy1rR25rLWd0dWprN2hHMGVvZ056GgA
2021/01/02 23:20:29 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVi10cGpxZnROMmQ2TjFZS1BBanBBNFV4NkpHTllzd2w4UHFieFFmODNwUXhYV0czdExYdW1ZTGNxY2wzeGN1Sk52QWd5QjdXLVBRSkppWGVMbHItQ0h5Y0dRY0d1WmtyYV9HRDVpZUs5ZVZCU2ZvRFp0cG5BdGRJbjNJbHBxb052a3RUQnN5NnlyTDFJSUd1NUs0MlZxTjhvUWF5GgA
2021/01/02 23:20:30 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVi10cGpxZnROMmQ2TjFZS1BBanBBNFV4NkpHTllzd2w4UHFieFFmODNwUXhYV0czdExYdW1ZTGNxY2wzeGN1Sk52QWd5QjdXLVBRSkppWGVMbHItQ0h5Y0dRY0d1WmtyYV9HRDVpZUs5ZVZCU2ZvRFp0cG5BdGRJbjNJbHBxb052a3RUQnN5NnlyTDFJSUd1NUs0MlZxTjhvUWF5GgA
2021/01/02 23:20:30 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVndNT0FRNzR4bWFWN1hQNHhTN2FqaVJlZDltUTJGd3RNQUF2clZ0WHpSTVhTcUpLSnRQTW0yd2czWEtPazBfbDV2bTQxb01JTFZaamFGOHRLS3VHQ1dDMFZHWElHTlVFbEMxemV2YXloeEtYV3V0N0xScUtTVkp2T3hSUm15NGFHcjkyVGtha285VGJmQWkzZmU2WGRNanluS050GgA
2021/01/02 23:20:30 [DEBUG] POST https://photoslibrary.googleapis.com/v1/mediaItems:batchCreate?alt=json
2021/01/02 23:20:30 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjl3RloyS1k5YnR6VmhpaUdXcjl4VGdJQ2hsTU5CY1BKWUlkRVNwVTZUNExLUWVKRHpGVzRiWXFEc3g5YUxISmJzZ0JtdVpMS3hmWFN5dGxkUmlqNkxPcTB5NzlCRFFQNmt2Zm1mNU9lbTYwNkExRXEzZ0RBZy1PZkNodmtuQnNfc1B1TWZNT1dYcGVvbkw4TVNRRllmUFFNTE1VGgA
2021/01/02 23:20:30 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVndNT0FRNzR4bWFWN1hQNHhTN2FqaVJlZDltUTJGd3RNQUF2clZ0WHpSTVhTcUpLSnRQTW0yd2czWEtPazBfbDV2bTQxb01JTFZaamFGOHRLS3VHQ1dDMFZHWElHTlVFbEMxemV2YXloeEtYV3V0N0xScUtTVkp2T3hSUm15NGFHcjkyVGtha285VGJmQWkzZmU2WGRNanluS050GgA
2021/01/02 23:20:30 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVndNT0FRNzR4bWFWN1hQNHhTN2FqaVJlZDltUTJGd3RNQUF2clZ0WHpSTVhTcUpLSnRQTW0yd2czWEtPazBfbDV2bTQxb01JTFZaamFGOHRLS3VHQ1dDMFZHWElHTlVFbEMxemV2YXloeEtYV3V0N0xScUtTVkp2T3hSUm15NGFHcjkyVGtha285VGJmQWkzZmU2WGRNanluS050GgA
2021/01/02 23:20:30 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjF0QTgteGpzS0o4eUJDSnRXYWttdnZKWmpMdGU3RTV3STJxNjBaVUFRZ2xoNzNtSU1EcW5lSWFkRGZZVkJVYWhOVmUyQzNuQ2hZREVPZE1uV1RiSkpfQklyd3gxMjFvRTI2NDEwRWxEZ2VHZXpwVkxtVzhnWVc2cUI1RXdXQzNuZDJhMlEyTy1rR25rLWd0dWprN2hHMGVvZ056GgA
2021/01/02 23:20:31 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjdOajk0WnlwRFBBMm5XcjZzLWJoYlk1ejRYS0xLSUpMcVRvZ0piTHlhdldGLUFWOFJMYjdzNXQ2dnV6VjhPU2l3dW1OQ0ZDZzFmS2FUS19HV0wzTGp1YXBJMUQ1dGR0YTIwVWtXc1Q5WWEtd0NOX2g4SHBRS1NtTUw5Q25MX0ozaHJJdHIwRl80alA3X0pSa3NKeGRsRG9KVGI5GgA
2021/01/02 23:20:31 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjF0QTgteGpzS0o4eUJDSnRXYWttdnZKWmpMdGU3RTV3STJxNjBaVUFRZ2xoNzNtSU1EcW5lSWFkRGZZVkJVYWhOVmUyQzNuQ2hZREVPZE1uV1RiSkpfQklyd3gxMjFvRTI2NDEwRWxEZ2VHZXpwVkxtVzhnWVc2cUI1RXdXQzNuZDJhMlEyTy1rR25rLWd0dWprN2hHMGVvZ056GgA
2021/01/02 23:20:31 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjF0QTgteGpzS0o4eUJDSnRXYWttdnZKWmpMdGU3RTV3STJxNjBaVUFRZ2xoNzNtSU1EcW5lSWFkRGZZVkJVYWhOVmUyQzNuQ2hZREVPZE1uV1RiSkpfQklyd3gxMjFvRTI2NDEwRWxEZ2VHZXpwVkxtVzhnWVc2cUI1RXdXQzNuZDJhMlEyTy1rR25rLWd0dWprN2hHMGVvZ056GgA
[debug]  Successfully processing /mnt/photos/2004-11-21-1244-07/DSCN1133.JPG
[debug]  Worker #2 processing: /mnt/photos/2004-11-21-1244-07/DSCN1134.JPG
2021/01/02 23:20:31 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjl3RloyS1k5YnR6VmhpaUdXcjl4VGdJQ2hsTU5CY1BKWUlkRVNwVTZUNExLUWVKRHpGVzRiWXFEc3g5YUxISmJzZ0JtdVpMS3hmWFN5dGxkUmlqNkxPcTB5NzlCRFFQNmt2Zm1mNU9lbTYwNkExRXEzZ0RBZy1PZkNodmtuQnNfc1B1TWZNT1dYcGVvbkw4TVNRRllmUFFNTE1VGgA
2021/01/02 23:20:32 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjl3RloyS1k5YnR6VmhpaUdXcjl4VGdJQ2hsTU5CY1BKWUlkRVNwVTZUNExLUWVKRHpGVzRiWXFEc3g5YUxISmJzZ0JtdVpMS3hmWFN5dGxkUmlqNkxPcTB5NzlCRFFQNmt2Zm1mNU9lbTYwNkExRXEzZ0RBZy1PZkNodmtuQnNfc1B1TWZNT1dYcGVvbkw4TVNRRllmUFFNTE1VGgA
2021/01/02 23:20:32 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>pcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjl3RloyS1k5YnR6VmhpaUdXcjl4VGdJQ2hsTU5CY1BKWUlkRVNwVTZUNExLUWVKRHpGVzRiWXFEc3g5YUxISmJzZ0JtdVpMS3hmWFN5dGxkUmlqNkxPcTB5NzlCRFFQNmt2Zm1mNU9lbTYwNkExRXEzZ0RBZy1PZkNodmtuQnNfc1B1TWZNT1dYcGVvbkw4TVNRRllmUFFNTE1VGgA
2021/01/02 23:20:33 [DEBUG] POST https://photoslibrary.googleapis.com/v1/mediaItems:batchCreate?alt=json
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x872cbc]

goroutine 69 [running]:
github.com/gphotosuploader/google-photos-api-client-go/v2/media_items.GooglePhotosRepository.CreateManyToAlbum(0xa1fae0, 0xc0010b5110, 0xa219a0, 0xc000024090, 0xc0001374f0, 0x4c, 0xc0010ad200, 0x1, 0x1, 0x40fc18, ...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/media_items/photoslibray_repository.go:62 +0x31c
github.com/gphotosuploader/google-photos-api-client-go/v2/media_items.HttpMediaItemsService.CreateManyToAlbum(...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/media_items/service.go:52
github.com/gphotosuploader/google-photos-api-client-go/v2/media_items.HttpMediaItemsService.CreateToAlbum(0xa21be0, 0xc001bda630, 0xa219a0, 0xc000024090, 0xc0001374f0, 0x4c, 0xc000bc6000, 0x213, 0xc0002d44b0, 0x2b, ...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/media_items/service.go:44 +0x108
github.com/gphotosuploader/google-photos-api-client-go/v2.Client.UploadFileToAlbum(0xa236c0, 0xc001bce320, 0xa23780, 0xc001bda640, 0xa17a00, 0xc00110de80, 0xa219a0, 0xc000024090, 0xc0001374f0, 0x4c, ...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/client.go:41 +0x16f
github.com/gphotosuploader/gphotos-uploader-cli/internal/task.(*EnqueuedUpload).addMediaToAlbum(0xc000597000, 0xc0002d44b0, 0x2b, 0x0, 0x0, 0xc0068c5e90, 0xc0068c5e90)
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/task/upload.go:70 +0xb3
github.com/gphotosuploader/gphotos-uploader-cli/internal/task.(*EnqueuedUpload).Process(0xc000597000, 0xc0002d44b0, 0x2b)
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/task/upload.go:38 +0x53
github.com/gphotosuploader/gphotos-uploader-cli/internal/worker.(*Worker).Start.func1(0xc00019e6e0)
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/worker/queue.go:139 +0x357
created by github.com/gphotosuploader/gphotos-uploader-cli/internal/worker.(*Worker).Start
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/worker/queue.go:127 +0x3f
```

Failed again -- even quicker (log below). However, it turns out that DSCN4527.JPG is corrupt -- this is the ZIP of that image. It consists entirely of NULLs! 
[DSCN4527.zip](https://github.com/gphotosuploader/gphotos-uploader-cli/files/5762009/DSCN4527.zip)


```
[debug]  Successfully processing /mnt/photos/2004-11-21-1244-07/DSCN1134.JPG
[debug]  Worker #1 processing: /mnt/photos/2008-01-29-2017-10/DSC_0027.JPG
2021/01/03 17:34:01 [DEBUG] POST https://photoslibrary.googleapis.com/v1/mediaItems:batchCreate?alt=json
[debug]  Successfully processing /mnt/photos/2008-01-29-2017-10/DSC_0027.JPG
[debug]  Worker #1 processing: /mnt/photos/2003-09-07-1537-26/DSCN4527.JPG
2021/01/03 17:34:02 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=
2021/01/03 17:34:02 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEqIDQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnlWdkEyR1FNenJWSmhGSkNJd2lzMklRWWRIRGNCVnBfY0pWN2h4cHhYREtTTWZ2bGt0VGg1clQ1STJYbDJqOGZpdE1sLVpkWEE2ZGtMbF91NGtkUHNFaXF6NGw5d0VCeGZ3UFY1YlNMWXVlTy1yNXo1S01qeWJ4Yzlkc0QtdmNiT2F6Mm5wMkkyRUxtSmMwOWQzTHBnMWtfdTlld0dGajlGV2xWczZwYmVpRlRfUE13cjgzYVphYlV4d1dNMWVCb1NyTzlqZ1M1QVk3Z2NubGVtVVhldmlVRjJ2Q0FYUVo1eHJ3dWJoRFV0MktKdW5ySDl2VWw5U2ZPNGYzU2l3RGlPT1ZacDV0d3kwOGpIT3VtbmYzS2FkUXlETGE0RUN0Y1ZZMFgwdDNMT3BDd011TEhPMVNzOVlsNmpSd2NFLTBTQ0V5VjdVbm9jSjMzQWJsZ21HN0pabzktZmYzdldRUVJzcWF1ekU2UDlYV05DVkdEQRoA
2021/01/03 17:34:04 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEqIDQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjh6TzZaOVZfMlFjcnBLYkQ5X2lTTF9MRGZhTUJESWxSMTlmMkFSeW1vbUpya1VtS3VfMFRDdEJyaXhOM21JQVdTcERkOUlVRlFuYUlMUEYwZUtPRGR5d1VvZ2Z4ZkJtUjhUTHNSSHZhZUl5c200ZmVVak04U2VkbnBpTTJxb1dETXRTRzUtbUxKRG11bTNpblJ0dVFUbzV3cW5NX0pXOW5nVHVSb3JCU1pCTU9BVGN0UjNHM081MG9Vam5na3Z4cEN5TkhRNldraG9fRTJjZm1wMzZyWE55aFFwbW12ekYwWGpLdGdocjlOUjJ1LTkwX1Q3OE1SeUtKcVh2ZmE3UEpIS0kxYkFLSm1JNi00d1JndHlwaXVaajBsdjRoRG5xSFR2MjkyQ2liSTZlTXJsR0NDU3VpNU84cmdoQ0xXWllFcUNGMnZRbFdLNEFsX04xTGpNRG1wQjBpT0t2ekpUMVN5ZWdpX2E1ZVZrZmVUdkpzZxoA
2021/01/03 17:34:04 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEqIDQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVi1Lak1MTFpPOTZzYjRubXhiU2d0Ujc4UHFZczdoWW5kOGVobHBqZm5ENEVyZk1YdkhGeXhQWGJHSGtCTVZRQXI0ZDc0cWtGdUE0Z1FTRnE2Q0treG44U1VQVzczS05iT0tyWUlxRjhRTXZITmdNdFNkNGJNalZTMUk0MW1RS1dCbkxoTHNYVVo1M0s3VjVTOGZUZXVPRjFsVzFqRUsxVDc4Y2NPNU9pVThzWGFsTE14cnRMV01ldDlrbzI5UW1VaEZvdmtHVDZLcW9IckVQOHNYMVZUS0JwWW5MTGFhMUxJcmFnME5CWDdqWl92OXBKeVZKX3doVmhkQ2ZGTFVlblRmbjlCMmp5aFFMRmgzTkdfMHZFVjRXOFFfRzlRWmNhcGZzUEhQVlZjWnJheTMwSXJFSkJtWmJpQjJkVHg2M3JRamRVMy1ET19fZE5GRElqWmt3T0wxVmg5dXU3Tk5nNzBsaHFLMWx3cXY1VjJpeXkwQRoA
2021/01/03 17:34:05 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEqIDQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjJlSWVqRmlOQ2RvOVhKM3NfMzFla21ucDlTb2h2ZHV0Ukk2dnAyejlxczdRT1huOFZiXzRERVdaQ29jUFJ4N3M0OUJXX1h6V3I5YTRrd3hkNFA3QnFnR1FSRzU0aWdiSFl1d25qck1Ub24tNThDYmxMaW4yWVR1R3lDc1F5Mk8xSkFQYUZ5VUcxdzI1Qzdva1RMbmtqMFRzQ0ZSMDZianMwU0ozUF9oRTNvc3dpcEoxeEwzM0VsaXpmUDFfakg3a3diVllSUTQ3NnQycGZTNnhVQkZhVHh2NUtvdnk5eGc3bWJ3Uk9lTTFVV0RwZnI2bjNWemxubm9QS2pxbG5pQnZhYnVqeEpxZzlfTGpQVnFPOUdVQmU3SUIyakJjd2JYeDd2a25HUW9UMGhvUUpVaHhSTEFfOFF1SnlYVE51Rm1QMHlwREdYblFaY3hhWFlXSXVhWWFKeXZ3VE5uRGc0anNSa1ZsQkhKbTMzUVExQmt0dxoA
2021/01/03 17:34:06 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEqIDQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjB6VGxRZlA0OHpQZUktMkVabWowYU81ZVhSY25vLUxQbUFJbzVncGxGWWpuRExsUFdHNE5oc3hqZXFQRjlNVkZYalRaX1A1clNOb3lwQTF1c2hzdFdJUW1NenJ2RWV1RGRCWkI0OEM4TFkwWDJVZkw0QTBOY1N3U01Va1AyNjVtdjRadzhOYkJXa2thTnB1WWotQUZuV0d5WC1JdnNpSTl0a1NhT19KVXJrdnNJZVlpWlBwN3Y5X043QWpXWkpYS0p5UHlLdVlNLTNQX1RVNU1vRUozcXNqWi03bDdlckF0M05QbGdsX2FhcGdPYmltWWt2ZDRBT1NyN2M1NWl5MUxFZUFfUWVybDNSa0FwRmFzM2V0Um5femVLU3ZUTzhjTWZTekNXelk5WWpLZFh1OWpvVF9zR1VHcVhMSm80UVZ2QlZMVmlneEt5Yzl4c0stU1pLMmtFLTU2c3o0dUE1UjB6Y3l5VFZldjBrYXdnME9pZxoA
2021/01/03 17:34:07 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEqIDQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjJKbEs3NV9lU3k5NFhCMTQzaWVFcE5XN2hDQUQ0UjZ3dElza1lRYU5CQ2R6THJacjZqQl92M3NvMEpyRVFzQXZscVRQTmpyVURTMzhKSmlBeGJuQjR0ZTd4TkpsRVVxdGVtLU53MHNubEZpdzFNV20yQU9PSWlBVVZtWGp2WV8wUURzbF9Jd1Y3aEYtbG1XWklaZ0tjVFlQQkY3Z19HcU1pejZHRnJTb2wwLTlsRUlBYURBNDNsSDlxY3AxR1plaUVXYWNndzlBT0dhWDBud09pNE9lVVI0N0F5ckozdVR6Q0NoRUtlOXZJOVlNd3RyR0c3OHFQVzBsZDJkRzJsVk1KMjEwbzZVNTNfWDA3OHVWdFUyNlNISWtqTUMybGwyZXh0M3hsdDJRUEczeXFSWkxwbHpHbF9aUExfOUVuQjVLMUI2R1VtOHRySFpQaDdwNU5tM1Y5VjdRRXh2Z0RFcVVaWTRlQTlmd3FXd2syTXlHURoA
2021/01/03 17:34:08 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEqIDQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVl85V09ISVh3X3lpZkFybWlZNllJYjdZRGIwcGRPRC14NFcyRjFyY3NRQ1VMMXFTcnVkUG5ISDNJMUNiU2dFdGpJNXJmR2tDQkNGd3RoSW41RFQxNUZpT2pHOXV4WEZuZ28wNkM1Nkk3eU85Z3REMWJZVGxNSjcxSXd4UFZiMFNBNEFIeUpMRHRTWDBZelBONHVncTFYZk1CeC1MSmlGYlRHb3ExN1VVODFha2JGZjVPWlplYlFMcjVyVG5VU2haZ0YxWWtBN0ZkWlB1OVJ1c2hFQWVyTFV6b2ZfdFNacFBMTXdSWG1NQWZNa09hbmNTaDdIY0lCWEFla3BRUFZUR2N3UmFhSGlFYXhtMDc2TGFMQUZSeVdTdlc0dDcwcUpIWTVzeWR0dXp5LThRdExNMTZnVnZGYU1ydkhXTThob2otOERrNF9CMHJUWnlvWXZxNWk2Q0p4Z2F6eE1SQ0trbWh0SHFaM3dYRFY3ME05N0FoURoA
2021/01/03 17:34:08 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVndIeEFxWFR5R0s4Y3lNbTNHTHNZdXl3V1Q1Ym1vcHR3c2ZuaFhKdnFWQUZIU0FyalNWd3RRRjhmR1BndlRpcEhSeTN1alFDeFZEWGtzVFpSbXJ6ODRSMmxDSGtUeW1zS21RcVRIaG9TMVNSQUxFeEZDcGVQSmpRaXBVbDY0SlgtUWprSktJWk4xTFFQOGZOSXJ2U2FzX2hTR1FzGgA
2021/01/03 17:34:09 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnhmZTNmcnRQMlVpaldjZVhiZ0ZyaF9NVEp4Ykd3NkhFU01BZ0dMQVlraGYtQWZkN2dLTEotb0VlV2hCTlR3TS03UmhndGw4dHFNT3BSZnpMR1o3bGJZQjB0VFBCVTJRc1lrY1RhNGJKT0xBV0pucGkzTWxqc0Ffb3dBUFlxOW03R0hMSU4wVklRRGl2ZDUxc2JZRlRXYnIzdjB2GgA
2021/01/03 17:34:10 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVi1QWTFKS0l6VW90c29FWkFsYTR3RUtXREZXSGtLZGpOS0tqWGFtUlY2NUR0RnVWLVJGeHRndVgxb2tUSklpNHBvU2dPMmw4cnFuZFAzZ1JwcmlxdXBBV2hMXzI5eFljNEtnWVBXNnY3VkZ4SDFFTU5nZnlYd0E5X29lRUViT2V5cXhJWl9uYm5xX3d6U1BSTVFXa256WGV0ZkxEGgA
2021/01/03 17:34:11 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjFLNGJ6Mm9yM3RpMjFxRzFuQzRELWlSblRVOU1UYXFtT1VPRksxR2NZR2EzSE5tbFZlbWNac1VpWEhucDd0eU05VHBlWmtKa3NfSDNHM2JzeVozdlFuRVJmSUpuejg3VG5KdU1QZmtMZVEtRGFXbVhKbUhUWnhJUU41REtVa1dLRDFQQTJHNFo3enI2bV9qWXRkWGl6QVVMak1IGgA
2021/01/03 17:34:11 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjZ6d3lQVjlneENmTFg4MnE4VjRoeElocUZjSkZ2S3E0c1loZ2pwOVFBbFpfMjBkSE15NHBpc3NjZGtzNWlxQV9aWTVZVmRyNUM1WjdNWXl4N2VoSmZPaURRMm1mb0JkNzJnRy15R1VCVjBsT1R1c2JocTNTeEM2dEhKYVRkbWNncHluMTJhaVJJYXh1d05sMG9KUUIwT1dKODd1GgA
2021/01/03 17:34:12 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjNTUWdNMHBuSG9Mdmd0WC1udUV3eHQyOGJvb1hzYWV6a1gxVUhKSkV3Vmx1cXM4RzQ4ejJHYUxIeks5WU00N1lPemNBbjBVNEtJU1pFSFNpQU5jM1FSUVpFWXFxYkpSSnpLTngxTFgxRVl6NzFjYVhLWXh3Nm1MZnMzbFJlTDV2V1pXSkRCNFJWU3FiVktoa3hGSzVFT3p1ODYzGgA
2021/01/03 17:34:13 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnlpdzZ6cXlSSWpmR21rRHZXeGpHbEVKSXRHcG5PRURaSHhOV2ZTN19HZ1g5TkE3RVZKUkRsRHBIRjh6SlBjSzNqR1JZVS1Cc05COHE0SUtuVngyRFVIcEZSZEZtVjltbEhUclhWS0FWZVVaUzRGT1pPVl9LaGtqa3RUcTdsV3VKa0lQOUU3WVlXbGFvZm1heTVVNktrUG5wX1Z4GgA
2021/01/03 17:34:14 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjhSbjV3TjUzWjlUcE43TEJFRHU1akdfREJaZDl4cFZiQmFGQWdTVWlKWkhCTmhCdXhTU0poeTFVOURqUVlicmRjSDNsbWc4ZnV2OENrWGgyeEQ4VUJmU29DVHV3clRNR040bW55bWo4N3Zlemo0VTZoQTFUUG5RZVdqS1BrQVRQUGJvaEhyVjF2S1RKTm11THoxV0hvNTZlNVpCGgA
2021/01/03 17:34:15 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnozS1k5Y29pRHBjRjVEUzJ4TThDN0ktazYxTWVIWUdqd0QyQk1lTzRhcFRnT0J6amlOUURqYzZYWjNUOThBbXBZR1ZyTG90OTZFWGVoaV9NeHhqYUtCZFU1cjU1akpXSjM4Nno3d3dKM3ZXMHdCLU9PVVFfUE11XzI2ZDB1bjlhcTFZYVNsbmNGaFIyVm0zeG9Ja1BZVndXYmJIGgA
2021/01/03 17:34:16 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVndlZEdsX3RmSGhLTlZJdnF1bUNzOXBFYjF6blhWaFl2bG5Oa21NSXdpRzlRdXpLNjdqYmJ2VWg3aDU5TF9mdmZQakw3Smc0SFg5ajZuZUZNWGltMmptWDZSSlo4ZEh5S3FIQWpYSjFWZDNmdkFfVmFRdEtJRHFEZXRRT0lrNkJic1NmVHBqWm4xZ2cyQUJDaUcyR1R4N0o0b3hDGgA
2021/01/03 17:34:17 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVndrRm15VjdobUJhLUt5VGs2a18tV0szaWRkS195VVEwZjRWaGkteDBKdTVtM2c0TnJvLWVPdUZpQ2tpWEdaNjBRWEtBbGd2azdKSjZGMWZCR1drZGJ4eWVtZVU2OTdwQkJmWFIwWDB5M1kwMTVHUzZhaEtjMWNTcnhEOXhqTlRQUHQ5REJPcUNZNTZjWU5ObjBkek9pdDlGVXAxGgA
2021/01/03 17:34:18 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVng3Q0pmbFIzNm1tZjVUV3B6WWN0TUMxS2h2a0JscmxieFVac24xOTcwdDVraEdBbEZSSExhZVRBbEM1QzVwQ25IVXFOMHc5ck5lTGZ5eE5OSmdTem9NQ1h3RmdtODgtYlc2ZVZaZXkxRzdvMXd4akpxc1lUNVplc0lnR2FpSlZYWkRac3oyTHQ1WllqR090c1ZRamVaMjV2MjI3GgA
2021/01/03 17:34:19 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjJoQUtINVZCd2lzbC04NzlWMWg4OFJfdTZSTTVrUXBzZ196QWZRU0luRlRmTi1TZjI3WjBGNlJMSGVJWWc5dzFIX1hDU0hTYjVzZEpNX09fcS1RZlZNb2Fmazc0ZHAxaG9aRUVEdGFucnZlSW1NOGc2TDd2TXRWMzVCOU12WFMtRnllbkpUdmVQRkhmODE1SElMMVRlazcwZm93GgA
2021/01/03 17:34:19 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVi0yeWZSMGZmaHQyTU1iTnZkUENtNEhUa1psUVVMT1NqZGpmTGpaN3hHT1FZdVUwRXc3NnZBa1BSMExVMHZrNXBfaGhDMjVuaU00LUlvcXlfTDZHeVJVVzdVMXBMOWdORFYwdS13Ukh3UGY5QVpyVlRGSjNMdkV2WUs2VV9VVDZiQnVncGRiVm5MM2dhOVB2VDJiS3Z4WVJsTS1UGgA
2021/01/03 17:34:20 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjByUXEydTloZWc1cjNMTVJVS3pEUjJDanBhQ1JYRVhmLUNxdWo5ODZ0TzlhT1UxZ05UN3pkY2tQUXVycC1Bbl9ITHVlMVpBUEt5Znl2ZWhLWUhxTVJTSXFibllaYjZjVENsSmZmM1Btd1psZGo5dnJVUTV2cDViLXBxTUN5UU4yZ3VmUFVBUk9uYWM2UGcySVhfVzloNDhLSk9jGgA
2021/01/03 17:34:21 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVi0zdlN4b2xFeGlrM04wS242aGg3X1J4aGpwWTlVOEE4VVRMMGtRdXFXdm15ZnotMGFvTC1hdExEeXotMFQ3MkYtZlRORlFMMFNyQi1nWmVxYm5HYnNUdzlLdUZZN2FLVTUyU1NxNWFQMlpWQmdjQ08yUEI4aUhwUWVsSnBid19HMTI2T2JDdVhkeXgxd1o0SFpoN2RPT0tHdF8yGgA
2021/01/03 17:34:21 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnd6RzUwRlJhWDE5eENraEllZ0tvV1lDV0h5VVFjWWtYYTlxMDhmSnhlWkVjVXlMbjdQTC1ldkhnRlBLNExDcXBIZEVOSDdtMEt1YVNTUTNpQmxVVXVBVTRDSW5iWHBNTUNiNTdPM01ORUc4UW9ZUExmc2F2U1RrWHdoQU9BQlFMc2d0ZGJwS2RvMlVaVUdBMHplSlM3TTkzanlXGgA
2021/01/03 17:34:22 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVnh3Y3BmYkZuT3RVWGJhNGtQZm9WZy14TGtJR21zOHRVTVZ1VS1Nd1J6bTdQR19hVm8taThjVTBVTFJTQzZ5VlN6amJoeU41ckx6OVJzQ0dMX1dfeWFwbl9PMGMyTUI5RlREc1lMd0JGSHkzUUxacFk3X2o0MGdOMnh0ZUlLRDJqczRIb2FSd0hwb004UnliZjZMeXZnTzBtRGtIGgA
2021/01/03 17:34:23 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVi10cGpxZnROMmQ2TjFZS1BBanBBNFV4NkpHTllzd2w4UHFieFFmODNwUXhYV0czdExYdW1ZTGNxY2wzeGN1Sk52QWd5QjdXLVBRSkppWGVMbHItQ0h5Y0dRY0d1WmtyYV9HRDVpZUs5ZVZCU2ZvRFp0cG5BdGRJbjNJbHBxb052a3RUQnN5NnlyTDFJSUd1NUs0MlZxTjhvUWF5GgA
2021/01/03 17:34:26 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVndNT0FRNzR4bWFWN1hQNHhTN2FqaVJlZDltUTJGd3RNQUF2clZ0WHpSTVhTcUpLSnRQTW0yd2czWEtPazBfbDV2bTQxb01JTFZaamFGOHRLS3VHQ1dDMFZHWElHTlVFbEMxemV2YXloeEtYV3V0N0xScUtTVkp2T3hSUm15NGFHcjkyVGtha285VGJmQWkzZmU2WGRNanluS050GgA
2021/01/03 17:34:26 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjF0QTgteGpzS0o4eUJDSnRXYWttdnZKWmpMdGU3RTV3STJxNjBaVUFRZ2xoNzNtSU1EcW5lSWFkRGZZVkJVYWhOVmUyQzNuQ2hZREVPZE1uV1RiSkpfQklyd3gxMjFvRTI2NDEwRWxEZ2VHZXpwVkxtVzhnWVc2cUI1RXdXQzNuZDJhMlEyTy1rR25rLWd0dWprN2hHMGVvZ056GgA
2021/01/03 17:34:27 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjl3RloyS1k5YnR6VmhpaUdXcjl4VGdJQ2hsTU5CY1BKWUlkRVNwVTZUNExLUWVKRHpGVzRiWXFEc3g5YUxISmJzZ0JtdVpMS3hmWFN5dGxkUmlqNkxPcTB5NzlCRFFQNmt2Zm1mNU9lbTYwNkExRXEzZ0RBZy1PZkNodmtuQnNfc1B1TWZNT1dYcGVvbkw4TVNRRllmUFFNTE1VGgA
2021/01/03 17:34:27 [DEBUG] GET https://photoslibrary.googleapis.com/v1/albums?excludeNonAppCreatedData=true&pageSize=50&pageToken=<removed>Bpcy5jb20vZ29vZ2xlLnBob3Rvcy5saWJyYXJ5LnYxLkxpc3RBbGJ1bXNSZXF1ZXN0EgQIMhgBEswBQUhfdVE0MTdyazJ4YkJidEVNNUxXTWFhOWhpSWJ6N0kwaEF1Yk9Fa3pGU1hZbDBYVjdOajk0WnlwRFBBMm5XcjZzLWJoYlk1ejRYS0xLSUpMcVRvZ0piTHlhdldGLUFWOFJMYjdzNXQ2dnV6VjhPU2l3dW1OQ0ZDZzFmS2FUS19HV0wzTGp1YXBJMUQ1dGR0YTIwVWtXc1Q5WWEtd0NOX2g4SHBRS1NtTUw5Q25MX0ozaHJJdHIwRl80alA3X0pSa3NKeGRsRG9KVGI5GgA
2021/01/03 17:34:29 [DEBUG] POST https://photoslibrary.googleapis.com/v1/mediaItems:batchCreate?alt=json
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x872cbc]

goroutine 68 [running]:
github.com/gphotosuploader/google-photos-api-client-go/v2/media_items.GooglePhotosRepository.CreateManyToAlbum(0xa1fae0, 0xc0003ee0c0, 0xa219a0, 0xc000024080, 0xc0006892c0, 0x4c, 0xc001857ac0, 0x1, 0x1, 0x40fc18, ...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/media_items/photoslibray_repository.go:62 +0x31c
github.com/gphotosuploader/google-photos-api-client-go/v2/media_items.HttpMediaItemsService.CreateManyToAlbum(...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/media_items/service.go:52
github.com/gphotosuploader/google-photos-api-client-go/v2/media_items.HttpMediaItemsService.CreateToAlbum(0xa21be0, 0xc0019f8840, 0xa219a0, 0xc000024080, 0xc0006892c0, 0x4c, 0xc007ae8000, 0x213, 0xc0002c8690, 0x2b, ...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/media_items/service.go:44 +0x108
github.com/gphotosuploader/google-photos-api-client-go/v2.Client.UploadFileToAlbum(0xa236c0, 0xc00187a380, 0xa23780, 0xc0019f8850, 0xa17a00, 0xc001e000c0, 0xa219a0, 0xc000024080, 0xc0006892c0, 0x4c, ...)
        /home/runner/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go/v2@v2.0.0/client.go:41 +0x16f
github.com/gphotosuploader/gphotos-uploader-cli/internal/task.(*EnqueuedUpload).addMediaToAlbum(0xc000576280, 0xc0002c8690, 0x2b, 0x0, 0x0, 0xc0075d1e90, 0xc0075d1e90)
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/task/upload.go:70 +0xb3
github.com/gphotosuploader/gphotos-uploader-cli/internal/task.(*EnqueuedUpload).Process(0xc000576280, 0xc0002c8690, 0x2b)
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/task/upload.go:38 +0x53
github.com/gphotosuploader/gphotos-uploader-cli/internal/worker.(*Worker).Start.func1(0xc0000207d0)
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/worker/queue.go:139 +0x357
created by github.com/gphotosuploader/gphotos-uploader-cli/internal/worker.(*Worker).Start
        /home/runner/work/gphotos-uploader-cli/gphotos-uploader-cli/internal/worker/queue.go:127 +0x3f
philip@saturn:~/bin$
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,6 +3,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/) and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## 2.0.1
+### Changed
+- Bump `google-photos-api-client-go` from `v2.0.0` to `v2.0.1`.
+### Fixed
+- Media item creation was failing when Google Photos was reporting errors on media creation. ([#262][i262])
+
+[i262]: https://github.com/gphotosuploader/gphotos-uploader-cli/issues/262
+
 ## 2.0.0
 > This is a **major upgrade** and it has some **non-backwards compatible changes**:
 > - `includePatterns` and `excludePatterns` configuration options has changed.

--- a/go.mod
+++ b/go.mod
@@ -12,7 +12,7 @@ require (
 	github.com/facebookgo/subset v0.0.0-20200203212716-c811ad88dec4 // indirect
 	github.com/facebookgo/symwalk v0.0.0-20150726040526-42004b9f3222
 	github.com/facebookgo/testname v0.0.0-20150612200628-5443337c3a12 // indirect
-	github.com/gphotosuploader/google-photos-api-client-go/v2 v2.0.0
+	github.com/gphotosuploader/google-photos-api-client-go/v2 v2.1.2
 	github.com/gphotosuploader/googlemirror v0.5.0
 	github.com/k0kubun/go-ansi v0.0.0-20180517002512-3bf9e2903213
 	github.com/mattn/go-colorable v0.1.2 // indirect

